lambda/apiprocessor/search.py

```diff
@@ -57,24 +39,16 @@ def search(request):
 
     if(keyword is not None):
         searchBody = {
-            "must": [
-                {
-                    "match": {
-                        "content": keyword
-                    }
-                }
-            ]
-        }
-
-        if(documentId is not None):
-            searchBody["must"].append(
-                {
-                    "match_phrase":
-                    {
-                        "documentId": documentId
-                    }
-                }
-            )
+                 "query" : {
+                   "match": { "content": keyword }
+                  },
+                  "highlight" : {
```

Comment:
We need to make sure that the old method of highlighting is removed

Reply:
The solution currently is using the front end itself for the actual highlighting happening on the UI.
This change helps us to replace the work done by conetextCount and context functions